src/DbUtils.php

```diff
@@ -1629,7 +1629,7 @@ public function formatUserName($ID, $login, $realname, $firstname, $link = 1, $c
 
         if (
             $ID > 0
-            && ((strlen($formatted ?? '') == 0) || $id_visible)
+            && ((strlen($formatted) == 0) || $id_visible)
```

Comment:
This suppose `$login` won't be null, are we absolutely sure of that?

Reply:
I hesitated quite a bit on this point. I supposed that a valid user would always have a login, but you are right, method can be misused.